### PR TITLE
Add `browse_github_actions()`

### LINKED ISSUE #1102: FR: Add `browse_github_actions()`
To expand the `browse_github_*()` helpers.

### PR DESCRIPTION
**Pull request by pat-s**
Fixes #1102

### CODE REVIEW

File: R/browse.R [hunk 1]
```diff
@@ -50,6 +50,11 @@ browse_github_pulls <- function(package = NULL, number = NULL) {
   pull <- if (is.null(number)) "pulls" else "pull"
   view_url(github_home(package), pull, number)
 }
+#' @export
+#' @rdname browse-this
+browse_github_actions <- function(package = NULL, number = NULL) {
```
Comment (jennybc) [inline R/browse.R:55]: It doesn't make sense to me for this function to have a `number` argument.

File: NEWS.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 * `use_tidy_ci()` is now deprecated in favour of `use_tidy_github_actions()` 
   (#1098).
+* New `browse_github_actions()` to open the Actions page of the respective repo on GitHub (#1102).
```
Comment (jennybc) [inline NEWS.md:5]: Please add your GitHub user name as seen in other bullet points (unless you explicitly don't want that).

**Comment by pat-s** [comment]
@jennybc Is there anything that needs to be changed to get this merged?

**Comment by jennybc** [comment]
Thanks!

**Pull request merged**